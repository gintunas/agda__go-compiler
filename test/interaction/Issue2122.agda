-- Andreas, 2016-07-29
--
--   agda --library-file=GARBAGE Issue2122.agda
--
-- should complain about non-existent library file.

-- This file is intentionally left empty.
