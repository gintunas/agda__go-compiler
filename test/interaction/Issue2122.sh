# Andreas, 2016-07-29, issue #2122

AGDA_BIN=$1

$AGDA_BIN --library-file=GARBAGE Issue2122.agda

# should complain about non-existent library file

# EOF
