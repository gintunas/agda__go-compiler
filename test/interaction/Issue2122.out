Error: Libraries file not found: GARBAGE
Run 'agda --help' for help on command line options.
